### PR TITLE
Amazon Sales Channel 4.1.0

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updates Amazon Sales Channel content for the 4.1.0 release.

## Affected documentation pages

- https://docs.magento.com/m2/ee/user_guide/sales-channels/asc/amazon-order-details.html
- https://docs.magento.com/m2/ee/user_guide/sales-channels/asc/amazon-orders-all.html
- https://docs.magento.com/m2/ee/user_guide/sales-channels/asc/amazon-store-dashboard.html
- https://docs.magento.com/m2/ee/user_guide/sales-channels/asc/cancel-unshipped-order.html
- https://docs.magento.com/m2/ee/user_guide/sales-channels/asc/fulfilled-by.html
- https://docs.magento.com/m2/ee/user_guide/sales-channels/asc/managing-orders.html
- https://docs.magento.com/m2/ee/user_guide/sales-channels/asc/ob-create-listing-rule.html
- https://docs.magento.com/m2/ee/user_guide/sales-channels/asc/order-settings.html
- https://docs.magento.com/m2/ee/user_guide/sales-channels/asc/product-listing-condition.html

## Additional information

src/_data/b2b/toc/marketing.yml 
- Added new amazon-orders-all topic
- Change Magento settings to Magento Admin Settings

src/_data/ce/toc/marketing.yml
- Added new amazon-orders-all topic
- Change Magento settings to Magento Admin Settings

src/_data/ee/toc/marketing.yml
- Added new amazon-orders-all topic
- Change Magento settings to Magento Admin Settings

src/sales-channels/asc/amazon-order-details.md 
Staging: https://docs.magedevteam.com/445/m2/ee/user_guide/sales-channels/asc/amazon-order-details.html
- Updated import order info
- Added note about dummy data from Amazon FBA orders
- Specified that table data is received from Amazon

src/sales-channels/asc/amazon-orders-all.md 
Staging: https://docs.magedevteam.com/445/m2/ee/user_guide/sales-channels/asc/amazon-orders-all.html
- New “View Amazon Orders” topic to document updated Recent Orders and new All Orders screens
- Included tips for some known issues that will be addressed in 4.2

src/sales-channels/asc/amazon-store-dashboard.md
Staging: https://docs.magedevteam.com/445/m2/ee/user_guide/sales-channels/asc/amazon-store-dashboard.html
- Removed recent orders details (moved to new topic)
- Adjusted second level bullets
- For recent orders, added references to new view orders topic
- Updated screen cap

src/sales-channels/asc/assets/amazon-orders-list-all.png
- New All Orders screen shot

src/sales-channels/asc/assets/amazon-recent-orders-imported.png
- Updated Recent Orders screen

src/sales-channels/asc/assets/amazon-store-dashboard.png 
- Updated dashboard screen shot

src/sales-channels/asc/cancel-unshipped-order.md 
Staging: https://docs.magedevteam.com/445/m2/ee/user_guide/sales-channels/asc/cancel-unshipped-order.html
- Corrected where orders will open, depending on import settings
- specified that returns and exchanges are handled in Amazon
- Specified that order import must be enabled

src/sales-channels/asc/fulfilled-by.md
Staging: https://docs.magedevteam.com/445/m2/ee/user_guide/sales-channels/asc/fulfilled-by.html
- Added note about dummy data for FBA orders

src/sales-channels/asc/managing-orders.md 
Staging: https://docs.magedevteam.com/445/m2/ee/user_guide/sales-channels/asc/managing-orders.html
- Removed all references to recent orders or viewing orders (and screen cap), moved this info to the new view orders topic
- added reference to new all orders topic
- Clarified some points of with and without import enabled
- Added scenarios that block order creation
- Added order import process table

src/sales-channels/asc/ob-create-listing-rule.md 
Staging: https://docs.magedevteam.com/445/m2/ee/user_guide/sales-channels/asc/ob-create-listing-rule.html
- Added note and links to view orders and manage orders
- Added order information note

src/sales-channels/asc/order-settings.md 
Staging: https://docs.magedevteam.com/445/m2/ee/user_guide/sales-channels/asc/order-settings.html
- Removed old order creation/blocker note
- Corrected note about recent which order number appears in the table and where Magento order number appears
- Added reference to managing orders topic

src/sales-channels/asc/product-listing-condition.md 
Staging: https://docs.magedevteam.com/445/m2/ee/user_guide/sales-channels/asc/product-listing-condition.html
- moved note about renewed products

### CODE REVIEW

File: src/sales-channels/asc/amazon-orders-all.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+1. Click **All Orders** in the _Recent Orders_ section.
+
+1. To narrow the list or search for a specific order number, complete the **Filter by** parameters and click **Apply Filters**.
+
+1. To view order details, click the Amazon order number in the _Order Number_ column.
+
+   Depending on your [order import setting]({% link sales-channels/asc/order-settings.md %}), the order details will open in Amazon or in the [Magento orders workflow]({% link sales/orders.md %}).
+
+{:.bs-callout .bs-callout-tip}
+A few notes about the orders view:<br/>- If you have multiple Amazon store integrations, you may need to refresh your page view when switching between store views. This updates both the orders list and the pagination views for the current store.<br/>- When sorting by column, the sort only applies to the current view orders list. This will be addressed in a future release.<br/>- Depending on the width of your view window, you may see overlapping text in the columns. Widen your window view to expand the columns for the text to wrap.
```
Comment (leslietilling) [inline src/sales-channels/asc/amazon-orders-all.md:65]: We are, unfortunately, forced to use <br/> in table cells but should avoid elsewhere. For a multi-line note, use `<div class="bs-callout-info" markdown="1">` and then format as you usually would.

File: src/sales-channels/asc/cancel-unshipped-order.md [hunk 2]
```diff
@@ -25,4 +30,4 @@ Amazon orders can only be canceled if they are in an `Unshipped` status. If the
 
     The cancellation notification will be sent to your Amazon Seller Central account, and the customer associated with the order will be notified. The status of the corresponding Magento order will change to `Complete`.
 
-1. Click **Cancel Order** in the header bar. The order is canceled, and the status is updated to `Canceled`.
+1. Click **Cancel Order** in the header bar. The order is canceled, and the status is updated to `Canceled` in the order details.
```
Comment (leslietilling) [inline src/sales-channels/asc/cancel-unshipped-order.md:33]:
```suggestion
1. Click **Cancel Order** in the header bar.

    The order is canceled, and the status is updated to `Canceled` in the order details.
```

File: src/sales-channels/asc/managing-orders.md [hunk 3]
```diff
@@ -3,49 +3,49 @@ title: Manage Orders and Fulfillment
 ---
 
 
-Order information received from Amazon Seller Central shows in the _Recent Orders_ section of the [store dashboard]({% link sales-channels/asc/amazon-store-dashboard.md %}).
+You can view your Amazon order information, as received from Amazon, in the _Recent Orders_ section of the [store dashboard]({% link sales-channels/asc/amazon-store-dashboard.md %}) or on the _Amazon orders_ page (also referred to the _All Orders_ view).
```
Comment (leslietilling) [inline src/sales-channels/asc/managing-orders.md:6]:
```suggestion
You can view your Amazon order information, as received from Amazon, in the _Recent Orders_ section of the [store dashboard]({% link sales-channels/asc/amazon-store-dashboard.md %}) or on the _Amazon orders_ page (also called the _All Orders_ view).
```

File: src/sales-channels/asc/amazon-orders-all.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+---
+title: View Amazon Orders
+---
+
+
+There are 2 ways to view your Amazon orders, _Recent Orders_ and _All Orders_.
```
[the remark this answers is not in the document]
Comment (hickskenh) [reply src/sales-channels/asc/amazon-orders-all.md:6]: revised in follow-up commit

File: src/sales-channels/asc/amazon-orders-all.md (hunk 1 above)
Comment (hickskenh) [reply src/sales-channels/asc/amazon-orders-all.md:65]: fixed in follow up commit

File: src/sales-channels/asc/cancel-unshipped-order.md (hunk 2 above)
Comment (hickskenh) [reply src/sales-channels/asc/cancel-unshipped-order.md:33]: fixed in followup commit

File: src/sales-channels/asc/managing-orders.md (hunk 3 above)
Comment (hickskenh) [reply src/sales-channels/asc/managing-orders.md:6]: fixed in followup commit

**Comment by hickskenh** [comment]
running tests

**Pull request merged**